### PR TITLE
OCPBUGS-17199: remove revision stability check from bootstrap complet…

### PR DESCRIPTION
**Pull request by tjungblu**
…eness

### CODE REVIEW

File: pkg/operator/ceohelpers/bootstrap.go [hunk 1]
```diff
@@ -182,20 +182,14 @@ func IsBootstrapComplete(configMapClient corev1listers.ConfigMapLister, staticPo
...
 	}
-	for _, curr := range status.NodeStatuses {
-		if curr.CurrentRevision != status.LatestAvailableRevision {
-			klog.V(4).Infof("bootstrap considered incomplete because revision %d is still in progress", status.LatestAvailableRevision)
-			return false, nil
-		}
-	}
 
 	// check if etcd-bootstrap member is still present within the etcd cluster membership
 	membersList, err := etcdClient.MemberList(context.Background())
```
Comment (tjungblu) [inline pkg/operator/ceohelpers/bootstrap.go:195]: alternatively, maybe smarter is to swap the order of the checks.

If we observe the etcd-bootstrap node is not there anymore, bootstrap is completely done and we don't need to check revisions anymore.

**Comment by tjungblu** [comment]
might still be the better option, I'm currently just figuring out why we check the revisions in the first place

**Review by geliu2016 — APPROVED** [review]
/label cherry-pick-approved

**Comment by hasbro17** [comment]
Given that we have other checks that tell us when bootstrapping is complete i.e checking the `kube-system/bootstrap` configmap and querying for the `etcd-bootstrap` member, I think we can remove the check around whether a revision rollout is in progress since that's a common operation and we would have a lot of false positive for wherever we are using `IsBootstrapComplete()` (`clusterMemberRemovalController` and `EtcdEndpointsController`) it seems.

/lgtm

**Comment by tjungblu** [comment]
Thanks @hasbro17 

/hold cancel

**Pull request merged**